### DISCUSSION TITLE
Unable to read Energy data over mqtt

### DISCUSSION THREAD

**Discussion by RudiSeitz** [arendst/Tasmota#20209] [Q&A]
After upgrading from 13.1.x to 13.2.x i can only read Energy Data (yesterday, actual voltage, ...) after push button "restart" (on web). I can read this data exactly one times. After a successful reading, there are no more anwers - until I press the button again.
I use this device (from amazon) "Tasmota Steckdose NOUS A1T"

**Comment by barbudor** [comment]
Could you please set `WebLog 3` and `teleperiod 60` and provide the console logs since restart for 5 minutes ?
There should be ~5 SENSOR message in the logs
